content/constants.js

```diff
@@ -43,10 +43,13 @@ if (CONTENT_TRANSLATED_ROOT) {
   REPOSITORY_URLS[CONTENT_TRANSLATED_ROOT] = "mdn/translated-content";
 }
 
+const FORBIDDEN_URL_SYMBOLS = ["\n", "\t"];
```

Comment:
If it's only used in one single file (i.e. `content/redirect.js`) feel free to put it there. And the day this constant is needed by multiple modules, then we can centralize it.